tools/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/pingcap/tidb/_tools
+
+go 1.12
+
+require (
+	github.com/golangci/golangci-lint v1.16.0
+	github.com/mgechev/revive v0.0.0-20190505013521-22b849f28677
+	github.com/pingcap/failpoint v0.0.0-20190430075617-bf45ab20bfc4
```

Comment:
we got different `failpoint` version between `failpoint-ctl` and failpoint runtime used in lightning code.